src/Inkluzitron/Contracts/IScheduledTaskHandler.cs

```diff
@@ -6,6 +6,7 @@ namespace Inkluzitron.Contracts
 {
     public interface IScheduledTaskHandler
     {
+        Task InitializeAsync() => Task.CompletedTask;
```

Comment:
Neříkal jsi, že ti to je proti srsti?

Reply:
Jo, ale převážilo to přidávání dalšího interface. Možná si zvykám :pepeLa: